packages/mui-material/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -488,7 +491,10 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(inProps, ref)
     swipeInstance.current.startX = currentX;
     swipeInstance.current.startY = currentY;
 
-    setMaybeSwiping(true);
+    flushSync(() => {
+      setMaybeSwiping(true);
+    });
+
     if (!open && paperRef.current) {
```

Comment:
I assumed that here `paperRef.current` will be defined for sure because of the `flushSync` before, which would change the `setMaybeSwiping` state, which is used for setting the `open` prop on the Drawer, and hence the modal. However, this is not the case, so I needed to add the `keepMounted` prop on the `ModalProps`, to be sure that the ref would be assigned to a node.

Reply:
We use `ModalProps={{ keepMounted: true }}` anyways at the moment